test/e2e/run.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# This script runs e2e tests on the AWS Load Balancer Controller
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo "Running AppMeshController integration test with the following variables
+KUBE CONFIG: $KUBE_CONFIG_PATH
```

Comment:
any reason to not use KUBECONFIG?

Reply:
We do specify $KUBE_CONFIG_PATH when triggering ginkgo tests . This KUBE CONFIG is only for logging information.